packaging_automation/publish-packages.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
```

Comment:
btw, can we please add `publish-packages.py` script in another pr ? since it doesn't seem be related with prepare release phase